packages/components/src/input/Input.tsx

```diff
@@ -40,7 +41,7 @@ const InputBaseComponent: React.StatelessComponent<InputBaseProps> = props => {
     <div className={className}>
       {icon ? <InputIcon>{icon}</InputIcon> : null}
       <input {...inputProps} onChange={cb} ref={inputRef} />
-      <InputLabel>{label || props.placeholder}</InputLabel>
+      {noLabel ? null : <InputLabel>{label || props.placeholder}</InputLabel>}
```

Comment:
does `!noLabel && (<InputLabe...` work? might be more readable than the ternary operator